src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -1102,6 +1117,25 @@ def call(
             encoder_attentions=inputs["encoder_outputs"].attentions,
         )
 
+    def serving_output(self, output):
+        return TFSeq2SeqModelOutput(
+            last_hidden_state=output.last_hidden_state,
+            past_key_values=tf.tuple(output.past_key_values)[1] if self.config.use_cache else None,
+            decoder_hidden_states=tf.convert_to_tensor(output.decoder_hidden_states)
+            if self.config.output_hidden_states
+            else None,
+            decoder_attentions=tf.convert_to_tensor(output.decoder_attentions)
+            if self.config.output_attentions
+            else None,
+            encoder_last_hidden_state=output.encoder_last_hidden_state,
+            encoder_hidden_states=tf.convert_to_tensor(output.encoder_hidden_states)
+            if self.config.output_hidden_states
+            else None,
+            encoder_attentions=tf.convert_to_tensor(output.decoder_attentions)
+            if self.config.output_attentions
+            else None,
```

Comment:
The formatter output is pretty awful there. Could we put those in intermediate variables just before to try and get something more readable?